packages/webpack-plugin/src/index.js

```diff
@@ -101,6 +101,19 @@ class LoadablePlugin {
   apply(compiler) {
     this.compiler = compiler
 
+    const { webpack } = compiler
+
+    const defs = {
+      'process.isBrowser':
+        compiler.options.target === 'web' ||
+        compiler.options.target === undefined,
+    }
+
+    // eslint-disable-next-line global-require
+    new ((webpack && webpack.DefinePlugin) || require('webpack').DefinePlugin)(
+      defs,
+    ).apply(compiler)
```

Comment:
with my suggestion this would become

```suggestion
  if (this.opts.serverSideModuleFederation) {
    // eslint-disable-next-line global-require
     new ((webpack && webpack.DefinePlugin) || require('webpack').DefinePlugin)({
       serverSideModuleFederation: true
     }).apply(compiler)
  }
```

Reply:
Nice job @bwhitty, I will test it here!! :rocket:

Reply:
@theKashey I am not being able to get the value of `serverSideModuleFederation` passed to DefinePlugin during the execution of the resolveProperty function, do you have some idea on how can I access this value?
I already tried with:
- process.serverSideModuleFederation
- global.serverSideModuleFederation
- only serverSideModuleFederation

Reply:
@ScriptedAlchemy I got these snippets from [NodeFederationPlugin](https://github.com/module-federation/universe/blob/1ca2c86a51167cc792d33342968a411dad879275/packages/node/src/plugins/NodeFederationPlugin.ts#L161-L172) do you have some idea why I can't get the serverSideModuleFederation during babel compilation execution?